rebasehelper/plugins/output_tools/__init__.py

```diff
@@ -51,7 +51,12 @@ def get_report_path(cls, app):
         return os.path.join(app.results_dir, REPORT + '.' + cls.EXTENSION)
 
     @classmethod
-    def prepend_results_dir_name(cls, *path_members):
+    def prepend_results_dir_name(cls, app, *path_members):
+        if app.conf.results_dir:
+            return os.path.join(app.conf.results_dir, RESULTS_DIR, *path_members)
+        elif app.conf.bugzilla_id:
+            subdir = os.getcwd().split('/')[-1]
```

Comment:
You should use `os.path.split`. But I don't like this special-casing too much. Knowing current working directory and the directory from where `rebase-helper` was started it should be possible to determine the proper path.

Reply:
It would be enough but can we somehow determine where `rebase-helper` started? I assume it would be necessary to pass it as an attribute of `Application`.